src/dali/plugin/pair_converter/ccxt_converter.py

```diff
@@ -0,0 +1,233 @@
+# Copyright 2022 macanudo527

```

Comment:
Rename this file just ccxt.py: it's package `dali.plugin.pair_converter` already clarifies this is a converter.